src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/Highlights.tsx

```diff
@@ -1,52 +1,47 @@
-import { Column, GridColumns, Text } from "@artsy/palette"
-import { DollarCircleIcon } from "Apps/Consign/Routes/MarketingLanding/Components/LandingPage/DollarCircleIcon"
-import { StarCircleIcon } from "Apps/Consign/Routes/MarketingLanding/Components/LandingPage/StarCircleIcon"
-import { GlobeNetworkCircleIcon } from "Apps/Consign/Routes/MarketingLanding/Components/LandingPage/GlobeNetworkCircleIcon"
-
+import { Box, Column, GridColumns, Text } from "@artsy/palette"
 import { ReactElement } from "react"
-import { Media } from "Utils/Responsive"
 
 const reasons = [
   {
     icon: (
-      <>
-        <Media greaterThan="xs">
-          <DollarCircleIcon height={50} width={50} />
-        </Media>
-        <Media at="xs">
-          <DollarCircleIcon height={30} width={30} />
-        </Media>
-      </>
+      <Box width={[30, 50]} height={[30, 50]}>
+        <img
+          src="https://files.artsy.net/images/DollarCircleIcon_1.svg"
```

Comment:
These ones are really out of line tonally. Curious about their origin story.

Also, evergreen tweet: you generally don't need to be using `Media`